api/types/kustomization.go

```diff
@@ -172,6 +172,31 @@ type Kustomization struct {
 	BuildMetadata []string `json:"buildMetadata,omitempty" yaml:"buildMetadata,omitempty"`
 }
 
+const (
+	deprecatedBaseWarningMessage           = "# Warning: bases was deprecated, Please use resources or try `kustomize edit fix`."
+	deprecatedPatchesJson6902Message       = "# Warning: patchesJson6902 is deprecated, Please use patches or try `kustomize edit fix`."
+	deprecatedPatchesStrategicMergeMessage = "# Warning: bases is patchesStrategicMerge, Please use patches or try `kustomize edit fix`."
```

Comment:
It looks like `patchesStrategicMerge` isn't handled by `kustomize edit fix` at the moment, so I think we should either wait for that change to go in first, or we need to change this error message to not mention kustomize edit fix yet.

Reply:
I fixed this message.

So, I add `edit fix` function with https://github.com/kubernetes-sigs/kustomize/pull/4733
If you have time, Could you watch this?